### DISCUSSION TITLE
my osu lazer aren't opening

### DISCUSSION THREAD

**Discussion by 12wer10** [ppy/osu#23902] [Q&A] [answered]
Actually I don't know what's the problem is. I'm using osu laser at MacBook Pro, my activity monitor says osu has no response. I did everything I could. Delete it, and re install, repeat. nothing help me. 
my problem is similar as this. https://github.com/ppy/osu/issues/8233 
similar as the link but osu don't make all of my applications stop. 

I left some log of here (~/Library/Application Support/osu/logs)
I hope that log are useful to solve this problem. 

Computer Specifications:
Model name: MacBook Pro 13
Processor: M2
Memory: 8 GB
macOS: Ventura 13.2.1 

[database.log](https://github.com/ppy/osu/files/11737043/database.log)
[input.log](https://github.com/ppy/osu/files/11737045/input.log)
[performance.log](https://github.com/ppy/osu/files/11737046/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11737047/runtime.log)

**Comment by peppy** [comment, marked as answer]
Please try deleting `framework.ini`. Your renderer is set to a mostly unsupported one.

**Reply by 12wer10** [reply]
the `framework.ini` are continues regenerating. 
is opengl isn’t suit for my mac? 
and how can I make `framework.ini` not regenerate?

**Reply by integerrr** [reply]
That file will regenerate on game launch if it's absent, you're deleting it only to reset the renderer settings, it doesn't mean it's a problematic file

**Reply by peppy** [reply]
Did the game start after you deleted it once?

**Reply by 12wer10** [reply]
yes, its works thanks 😊

**Reply by Wilsonrpg** [reply]
Where do you find the framework.ini?

**Reply by peppy** [reply]
It depends on your operating system. If you're on macOS, it should be at either

`~/Library/Application Support/osu` (new)
`~/.local/share/osu` (old)

You can hit cmd-shift-g in finder to paste one of the above locations.

**Reply by Wilsonrpg** [reply]
This worked perfect, thank you
